### ISSUE TITLE
Add shadows

### ISSUE THREAD

**Issue by dmarrazzo** [lupoDharkael/flameshot#313]
Please add an option to put under the drawing (box, arrows, etc) a black blurred shadow.
It seems just vanity, but it helps to distinguish the marker from the original shot.

**Comment by mmahmoudian** [comment]
@borgmanJeremy can this be also labeled as "Good first issue"?

**Comment by borgmanJeremy** [comment]
Yes I think those are all grouped together and can all be good first issue, I'll tag them.

**Comment by claytron5000** [comment]
From a duplicate issue:

One problem I have with Flameshot annotations is that they don't always stand out from the background. One solution to this would be to add drop-shadows, or the option to add drop-shadows to annotations.

Simple drop-shadows may be enough, or one could look to Skitch for inspiration and add white outlines + drop-shadows (at least on text).

**Comment by claytron5000** [comment]
I've been playing with adding drop-shadows to the application. I looked at `QGraphicsDropShadowEffect`, but I haven't been able to figure out how to apply to the painter. 
I've also played with the arrow, duplicating the parts, coloring them black,  and translating them. This seems like a fragile and tedious method. 

Any ideas how to effectively implement drop-shadows across the app? I was looking at this [ProxyStyle](https://stackoverflow.com/questions/5393462/qt-text-with-shadow) as a way to override text. Is it possible to do that on painted elements as well?

**Comment by Harliff** [comment]
+1 vote for this request!

The shadow is a very important feature that I really missed on the Linux desktop. When you post a white screenshot on a forum white page, you definitely want the screenshot to be separate from the white-colored forum page.

You can simply add a black pixel (or more black pixels) around the edges of the screenshot. Although it doesn't look as pretty as the shadow, it solves the main problem of separating the page from the screenshot.

**Comment by nckdryr** [comment]
+1 vote for request
